### ISSUE TITLE
Migrate documentation and schemas to catalogapi

### ISSUE THREAD

**Issue by Castronova** [I-GUIDE/catalog#121]
Move documentation and metadata schemas from data-catalog to catalogapi.

**Comment by pkdash** [comment]
@Castronova and @igarousi I assume we decided to rename the repo **catalogapi** to **catalog**. From the data-catalog repo it seems we just need to copy 2 folders -  **schema** and **examples** to the new docs folder in catalog repo. Is the main branch of data-catalog up-to-date or are we waiting for any PRs to be merged?

**Comment by Castronova** [comment]
@pkdash Yes, this is correct. I'm working with Irene to take care of the migration.

**Comment by horsburgh** [comment]
@Castronova and @igarousi - did this get done, or can I have @pkdash do this? I want to close out the issues tagged for this release.

**Comment by Castronova** [comment]
This is not complete and I would prefer that we take care in migrating our schemas over the the new repository. Specifically, we want to make sure that the we update, correct, and validate the schemas (and other docs) as we move them to the new repository. Me and @igarousi will prioritize this effort. For now, I think we should remove this ticket from the milestone.

**Comment by horsburgh** [comment]
@Castronova - I added this to the next milestone because I think we want to have the GitHub repositories fully sorted out as soon as we can so we don't have pieces spread all over.
